### ISSUE TITLE
Doge mod and Hardcore questing issues, post update. Minor hardcore questing bug

### ISSUE THREAD

**Issue by JurasTW** [xJon/The-1.7.10-Pack#1508]
After update Doge mod had a few issues i have noticed.
First of all it seemed that all Doge items disappeared from my existing world and needed to be crafted from scratch.
Second, quest "Power to the max" rewards, featuring Doge items has broken rewards. The text in the quest is the only hint that the reward is related to Doge
Also all Reward bag that should have given Doge items give no items. Reward "much wow" and reward "Random #15". 

My guess is that Doge mod item IDs changed or something?
Either way hardcore questing rewards were disappointing as they were. Receiving an empty reward is even more sad.
Solution would be to update the reward id's, i guess?

![2018-04-05_21 24 16](https://user-images.githubusercontent.com/38030680/38385218-fcc32e84-3919-11e8-9c97-e71e42722f88.png)
![2018-04-05_21 32 33](https://user-images.githubusercontent.com/38030680/38385219-fd130b52-3919-11e8-9138-3a58b14b260c.png)
![2018-04-05_21 35 19](https://user-images.githubusercontent.com/38030680/38385220-fd34dc3c-3919-11e8-8495-b24a943388ff.png)

**Comment by xJon** [comment]
Yes, the new Doge items should be updated in HQM and will be in the next update.
Weirdly enough the mod changed after its update.
Thank you for reporting!

**Comment by xJon** [comment]
**Also reminding myself to check the exploration backpack quest for bugs.**
